webhooks/webhooks.helper.ts

```diff
@@ -0,0 +1,300 @@
+        await this.clickCheckbox(idConst.allEventsCb);
+        await this.enterUrl(url);
+        await this.enterFilter(filter);
+        await this.enterName(name);
+        await this.clickOK();
+    }
+
+    static async EditFields(url: string, filter: string, name: string) {
+        await this.enterUrl(url);
+        await this.enterFilter(filter);
+        await this.enterName(name);
+        await this.clickOK();
+    }
+
+    static async verifyWebhookRecord(recordName: string) {
+        StepLogger.subStep(`Verify that ${recordName} is displayed under Webhook Table`);
+        await ExpectationHelper.verifyDisplayedStatus(
+            WebhooksPage.getWebhookRecord(recordName),
+            recordName)
+        ;
+    }
+
+    static async verifyWebhookRecordAndType(recordName: string, type: string) {
+        StepLogger.subStep(`Verify that ${recordName} is displayed under Webhook Table as ${type} type`);
+        await ExpectationHelper.verifyDisplayedStatus(
+            WebhooksPage.getWebhookRecordAndType(recordName, type),
+            recordName);
+    }
+
+    static async verifyWebhookRecordNotDisplayed(recordName: string) {
+        StepLogger.subStep(`Verify that ${recordName} is displayed under Webhook Table`);
+        await WaitHelper.waitForElementToBeHidden(WebhooksPage.getWebhookRecord(recordName));
+        await ExpectationHelper.verifyNotDisplayedStatus(
+            WebhooksPage.getWebhookRecord(recordName),
+            recordName);
+    }
+
+    static async clickRecordEditIcon(recordName: string) {
+        const editIcon = WebhooksPage.getRecordEditIcon(recordName);
+        StepLogger.subVerification(`Click Edit for ${recordName}`);
+        // normal click does not work for this element, so we use clickUsingJS
+        await ElementHelper.clickUsingJs(editIcon);
+    }
+
+    static async clickRecordDeleteIcon(recordName: string) {
+        const deleteIcon = WebhooksPage.getRecordDeleteIcon(recordName);
+        StepLogger.subVerification(`Click Delete for ${recordName}`);
+        // normal click does not work for this element, so we use clickUsingJS
+        await ElementHelper.clickUsingJs(deleteIcon);
+    }
+
+    static async deleteRecordAndOK(recordName: string) {
+        await this.clickRecordDeleteIcon(recordName);
+        await this.clickOK();
+    }
+
+    static async loginAndCreateWebhookRecord(url: string, filter: string, name: string) {
+        await ManuscriptLoginPageHelper.loginToManuscript();
+        await ManuscriptHomePageHelper.verifyPageLoaded();
+        await ManuscriptHomePageHelper.mouseHoverAvatarAndClickWebhooks();
+        await this.verifyPage();
+        await this.clickAddNewHook();
+        await this.enterRequiredFieldsAndOK(url, filter, name);
+        await this.verifyWebhookRecord(name);
+    }
+
+    static async verifyEditHookDialog() {
+        StepLogger.subVerification('Verify Edit Hook Dialog is displayed');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400130)
```suggestion
        StepLogger.subVerification("Verify Edit Hook Dialog is displayed");
```